questdb-rs/src/ingress/mod.rs

```diff
@@ -186,6 +126,10 @@ impl <'a> TableName<'a> {
 
         Ok(Self { name: name })
     }
+
+    pub unsafe fn new_unchecked(name: &'a str) -> Self {
```

Comment:
This fn signature is a bit misleading. 
Unsafe doesn't make much sense for &str. &str must always be valid UTF-8.

Reply:
Done.